src/app/Upload.svelte

```diff
@@ -1,4 +1,12 @@
 <script lang="ts">
+  import type {
+    SaveFileRequestPayload,
+    EventPayload,
+    CompileModelRequestPayload,
+    CompileModelResponsePayload,
+  } from "../common/events";
```

Comment:
Can we make this an import to `@common/events` instead?

```suggestion
  } from "@common/events";
```

This will require a change to the `tsconfig.json` to include `@common/` in the `baseURL`/`paths` config. Please see the root `tsconfig.json` for details!